models/quality_measures/intermediate/cbe0055_diabetes_eye_exam/quality_measures__int_cbe0055_numerator.sql

```diff
@@ -0,0 +1,238 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
+   )
+}}
+
+with denominator as (
+
+    select
+          patient_id
+        , performance_period_begin
+        , performance_period_end
+        , measure_id
+        , measure_name
+        , measure_version
+    from {{ ref('quality_measures__int_cbe0055__denominator') }}
+
+)
+
+, retina_test_code as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    From {{ref('quality_measures__value_sets')}}
+    where lower(concept_name) in  (
+          'diabetic retinopathy'
+        , 'ophthalmological services'
```

Comment:
This concept should be moved to the visit code logic in the denominator. You can confirm the codes in this concept are part of the "Patient encounter during the performance period" logic for the denominator in the measure definition.